### PR TITLE
Implement unlink in package installation

### PR DESCRIPTION
**Pull request by avermeer-tc**
##### SUMMARY
This implements the unlink option in package installations.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
junos_package

##### ADDITIONAL INFORMATION
This is particularly useful when trying to upgrade an EX2300 or EX3400. See https://supportportal.juniper.net/s/article/EX-Not-enough-storage-while-upgrading-Junos-EX2300-and-EX3400?language=en_US

This option is present in the Ansible Galaxy repository: https://galaxy.ansible.com/juniper/device

### CODE REVIEW

File: plugins/modules/junos_package.py [hunk 1]
```diff
@@ -52,6 +52,12 @@
       to the remote device prior to installing.
     type: bool
     default: no
+  unlink:
+    description:
+    - The I(unlink) argument is responsible for instructing the remote device to
+    remove the package after installation.
```
Comment (NilashishC) [inline plugins/modules/junos_package.py:58]: The indentation issue here is resulting in a traceback from collection_prep's doc generation tool. As a result, a lot of the RSTs are getting deleted. Could you please fix this and push the changes? Thank you.

File: plugins/modules/junos_package.py [hunk 2]
```diff
@@ -52,6 +52,12 @@
...
+    type: bool
+    default: no
```
Comment (NilashishC) [inline plugins/modules/junos_package.py:60]:
```suggestion
    default: false
```

Comment (NilashishC) [reply plugins/modules/junos_package.py:60]: We're trying to adhere to https://github.com/ansible-collections/junipernetworks.junos/issues/376.

**Comment by avermeer-tc** [comment]
@NilashishC I think my last commit addresses all your concerns.

File: plugins/modules/junos_package.py (hunk 2 above)
Comment (avermeer-tc) [reply plugins/modules/junos_package.py]: I've changed this to false

**Comment by rohitthakur2590** [comment]
@avermeer-tc  Could you resolve the current conflics, and also it seems docs file got removed with your PR.

**Pull request closed without merging**